File: Iniciante/1020 - Idade em Dias/Python/1020.py
def main():
    person_age = int(input())
    DAYS_PER_YEAR = 365
    DAYS_PER_MONTH = 30
    years, remaining_days = divmod(person_age, DAYS_PER_YEAR)
    months, days = divmod(remaining_days, DAYS_PER_MONTH)
    print(f"{years} ano(s)\n{months} mes(es)\n{days} dia(s)")


if __name__ == "__main__":
    main()

File: Iniciante/1020 - Idade em Dias/JavaScript/1020.js
function main() {
  const input =require("fs").readFileSync("/dev/stdin", "utf8");
  const personAge = parseInt(input);
  const DAYS_PER_YEAR = 365;
  const DAYS_PER_MONTH = 30;
  const years = Math.trunc(personAge / DAYS_PER_YEAR);
  const remainingDays = personAge % DAYS_PER_YEAR;
  const months = Math.trunc(remainingDays / DAYS_PER_MONTH);
  const days = Math.trunc(remainingDays % DAYS_PER_MONTH);
  console.log(`${years} ano(s)\n${months} mes(es)\n${days} dia(s)`);
}

main();
